src/App.css

```diff
@@ -36,3 +36,131 @@
     transform: rotate(360deg);
   }
 }
+
+.wrapper {
+  max-width: 1200px;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.progressBar {
```

Comment:
Make progress a block element, to not be behind images on JSX render

.output {
  display: block;
}